.circleci/config.yml

```diff
@@ -199,10 +199,8 @@ workflows:
     jobs:
       - test_ios:
           filters:
-            tags:
-              only: /^[0-9]+\.[0-9]+\.[0-9]+/
             branches:
-              ignore: /.*/
+              only: main
```

Comment:
I think this will break our tagged builds (releases).  Since the jobs `build_ios` and `build_android` depend on the testing jobs but the testing jobs no longer build on tags.  3 options I can think of:
1. Remove the test job dependencies from the build jobs (since every commit to main is getting tested anyway) - this is a little dangerous though because someone could theoretically create a tag before the tests have completed for a commit and introduce a breaking change.
2. Figure out a way to have the tests run on every commit and in the release builds (but seems wasteful).
3. Release every commit to main? (I kind of like the sound of this one)